core/vibes/soul/primitives/products-list/compare-drawer.tsx

```diff
@@ -1,11 +1,11 @@
 'use client';
 
 import { ArrowRight, X } from 'lucide-react';
+import Image from 'next/image';
```

Comment:
Are we going to use `next/image` for now until we decide on the `loader` pattern and do another performance pass later?

Reply:
No, this was an ovesight. Fixing and merging.